evap/evaluation/management/commands/send_reminders.py

```diff
@@ -33,14 +30,11 @@ def check_reminders(self):
             for course in Course.objects.filter(participants=recipient, state='inEvaluation').exclude(voters=recipient):
                 due_courses[course] = (course.vote_end_date - datetime.date.today()).days
             first_due_in_days = min(due_courses.values())
-            # sort courses by number of days left for evaluation and bring them to following format: [(course, due_in_days), ...]
+
+            # Sort courses by number of days left for evaluation and bring them to following format:
+            # [(course, due_in_days), ...]
             due_courses = sorted(due_courses.items(), key=operator.itemgetter(1))
 
             EmailTemplate.send_reminder_to_user(recipient, first_due_in_days=first_due_in_days, due_courses=due_courses)
-        logger.info("check_reminders finished.")
-
-    def handle(self, *args, **options):
-        if len(args) > 0 and args[0] == 'daily':
-            self.check_reminders()
-        else:
-            self.update_courses()
+        logger.info("send_reminders finished.")
+        logger.info("sent reminders to %s people." % len(recipients))
```

Comment:
use `logger.info("sent reminders to %s people.", len(recipients))`

...i wonder why landscape hasn't complained about that...